po/zanata.xml

```diff
@@ -4,101 +4,4 @@
   <project>dnf</project>
   <project-version>master</project-version>
   <project-type>gettext</project-type>
-  <locales>
-    <locale>af</locale>
-    <locale>am</locale>
-    <locale>ar</locale>
-    <locale>as</locale>
-    <locale>ast</locale>
-    <locale>bal</locale>
-    <locale>be</locale>
-    <locale>bg</locale>
-    <locale>bn</locale>
-    <locale map-from="bn_IN">bn-IN</locale>
```

Comment:
zanata does this automatically for long time already.

Actually there can not be different locale naming in Fedora, because locale names are standard. And actually `bn-IN` is invalid locale name.

```
[brain@x1carbon po]$ rpm -qf /usr/share/locale/bn-IN
file /usr/share/locale/bn-IN is not owned by any package
```